templates/test/ci/prow-user-assigned-identity/kustomization.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+namespace: default
+resources:
+  - ../../../flavors/external-cloud-provider
+  - ../prow/cni-resource-set.yaml
+patchesStrategicMerge:
+  - ../patches/tags.yaml
+  - ../patches/controller-manager.yaml
+  - ../patches/cluster-cni.yaml
+  - ../patches/apiserver.yaml
+  - patches/azure-machine-template.yaml
```

Comment:
we have a user-assigned identity flavor in the templates/ dir already which includes patches, wonder if we can reuse those since they seem to be doing the same thing?

Reply:
It seemed to me that we were keeping them separate from the test/ci templates. The test framework expects a local clusterctl repository that is currently populated just from there.